### ISSUE TITLE
NullpointerException State.Builder.getLocale()

### ISSUE THREAD

**Issue by joeabouassi** [Instabug/android-sample#67]
The app I was working on crashed in the background with the following stacktrace:

 java.lang.NullPointerException: Attempt to invoke virtual method 'android.content.res.Resources android.content.Context.getResources()' on a null object reference
                                                                        at com.instabug.library.model.State$Builder.getLocale(State.java:732)
                                                                        at com.instabug.library.model.State$Builder.build(State.java:687)
                                                                        at com.instabug.library.bugreporting.model.Bug$a.a(Bug.java:312)
                                                                        at com.instabug.library.bugreporting.a.a(LiveBugManager.java:67)
                                                                        at com.instabug.library.invocation.b.a(InvocationManager.java:301)
                                                                        at com.instabug.library.invocation.b.a(InvocationManager.java:232)
                                                                        at com.instabug.library.invocation.a.e.c(ShakeInvoker.java:36)
                                                                        at com.instabug.library.util.k.onSensorChanged(ShakeDetector.java:52)
                                                                        at android.hardware.SystemSensorManager$SensorEventQueue.dispatchSensorEvent(SystemSensorManager.java:583)
                                                                        at android.os.MessageQueue.nativePollOnce(Native Method)
                                                                        at android.os.MessageQueue.next(MessageQueue.java:323)
                                                                        at android.os.Looper.loop(Looper.java:143)
                                                                        at android.app.ActivityThread.main(ActivityThread.java:7224)
                                                                        at java.lang.reflect.Method.invoke(Native Method)
                                                                        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1230)
                                                                        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1120)

**Comment by joeabouassi** [comment]
This is happening a lot using library version 4.3.2

**Comment by reemalashry1** [comment]
Hey @joeabouassi we got another report of this this issue and we are looking into it.
I'll get back to you about it :)

**Comment by reemalashry1** [comment]
@joeabouassi  May I ask you if you're using Android studio 3 or if your app supports different languages?

**Comment by joeabouassi** [comment]
@reemalashry1 Thank you for your reply.

Im currently using Android Studio 2.3.3 and my app doesn't support different languages currently, only english.

**Comment by reemalashry1** [comment]
@joeabouassi 
We have fixed the issue and made a snapshot for you that has the fix, kindly use 4.3.2.5-SNAPSHOT

To use the snapshot, please add the following lines to your app-level `build.gradle` file:

```
dependencies {
 compile 'com.instabug.library:instabug:4.3.2.5-SNAPSHOT'
}
repositories {
   maven {
       //  add this only if interested in getting SNAPSHOT releases
       url "https://oss.sonatype.org/content/repositories/snapshots"
   }
}
```

**Comment by joeabouassi** [comment]
@reemalashry1 Thank you for the update, any idea when this fix will be added with the latest official sdk release ?

**Comment by reemalashry1** [comment]
@joeabouassi I don't have an ETA  :( but I'll definitely let you know when the release is out!

**Comment by joeabouassi** [comment]
@reemalashry1 no problem! Thanks

**Comment by reemalashry1** [comment]
@joeabouassi We have released the new version with the fix, please check it here: a8c20db3ba16c51e1c4f8278338dd5fcc9b80cd5

**Comment by joeabouassi** [comment]
@reemalashry1 Thanks for the update, that was fast!
